packages/wallet-contract/schema/v1/wallet-contract-documents.json

```diff
@@ -0,0 +1,55 @@
+{
+  "txMetadata": {
+    "type": "object",
+    "indices": [
+      {
+        "name": "ownerId",
+        "properties": [
+          {
+            "$ownerId": "asc"
+          }
+        ]
+      },
+      {
+        "name": "ownerIdAndCreatedAt",
+        "properties": [
+          {
+            "$ownerId": "asc"
+          },
+          {
+            "$createdAt": "asc"
+          }
+        ]
+      }
+    ],
+    "properties": {
+      "keyIndex": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "The index of the owners identity public key used to derive the encryption key.",
```

Comment:
The derivation index used to create the encryption key.

Reply:
This is a better description